numpy/_core/src/npysort/selection.cpp

```diff
@@ -28,6 +28,7 @@
 #include "simd_qsort.hpp"
 
 #define NOT_USED NPY_UNUSED(unused)
+#define DISABLE_HIGHWAY_OPTIMIZATION (defined(__arm__) || defined(__aarch64__))
```

Comment:
```suggestion
```
Based on the presented suggestion, there would be no purpose in keeping this #definition.